src/collections/programs/hacktoberfest-2020/index.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: "Hacktoberfest 2020"
+thumbnail: ./hacktoberfest2020.png
+link: "hacktoberfest/2020"
+published: true
+---
+
+import { ProgramsWrapper } from "../Programs.style.js";
+import Button from "../../../reusecore/Button";
+import hacktoberfest2020 from "./hacktoberfest2020.png";
+
+<ProgramsWrapper>
+
+<img src={hacktoberfest2020}/>
+<h2>Participate in Hacktoberfest 2020 with Layer5</h2>
+<div>
+    Hacktoberfest is open to everyone in the global community. Whether you’re
+      a developer, student learning to code, event host, or a company of any
+      size, you can help drive growth of open source and make positive
+      contributions to an ever-growing community. All backgrounds and skill
+      levels are encouraged to complete the challenge. Read more about
```

Comment:
There is no word space between "about Hacktoberfest". Please add that,

Reply:
This has been fixed